riposte/riposte.py

```diff
@@ -120,6 +122,10 @@ def prompt(self):
         """
         return self._prompt
 
+    @property
+    def splash(self):
+        return self._splash
```

Comment:
It seems to me that the banner doesn't need to be a `property`. It will be displayed once before starting the eval loop so there is no need to make it dynamic just like `prompt`. Am I missing something here?

Reply:
You're right, as there is not really any further context in which access would be required to the attribute